### PR TITLE
added runif and rnorm to function registry

### PR DESCRIPTION
**Pull request by jbloom22**
- added to expr doc
- added tests

### CODE REVIEW

**Comment by jbloom22** [comment]
These are especially useful for generating random quantitative phenotypes for examples and testing.

File: docs/HailExpressionLanguage.md [hunk 1]
```diff
@@ -50,8 +50,10 @@ Several Hail commands provide the ability to perform a broad array of computatio
     - toLong: `str.toLong`
     - toFloat: `str.toFloat`
 
- - Boolean coin flip:
-     - pcoin(p) -- returns `true` with probability `p`. `p` should be between 0.0 and 1.0.
+ - Random Booleans and doubles:
+     - pcoin(p) -- returns `true` with probability `p`. `p` should be between 0.0 and 1.0
+     - runif(max) -- returns a random draw from a uniform distribution on the interval \[0, max\]. `p` should be greater than 0.0.
```
Comment (cseed) [inline docs/HailExpressionLanguage.md]: `math.random` return something in `[0, 1)`, so it should say `[0, max)`.  The second sentence is wrong.  Also, if you do `runif(-5)` you get something in `(max, 0]`.  Finally, R's `runif(min, max)` returns something in `[min, max]`:

http://stat.ethz.ch/R-manual/R-devel/library/stats/html/Uniform.html

although note this remark, which I don't fully understand:

 > runif will not generate either of the extreme values unless max = min or max-min is small compared to min, and in particular not for the default arguments.

File: src/main/scala/org/broadinstitute/hail/expr/FunctionRegistry.scala [hunk 2]
```diff
@@ -279,4 +279,8 @@ object FunctionRegistry {
 
   register("pcoin", { (p: Float) => math.random < p })
   register("pcoin", { (p: Double) => math.random < p })
+  register("runif", { (max: Float) => max * math.random })
```
Comment (cseed) [inline src/main/scala/org/broadinstitute/hail/expr/FunctionRegistry.scala]: `math.random` is a double so this returns a `Double`.  Is that what you intend?

File: src/main/scala/org/broadinstitute/hail/expr/FunctionRegistry.scala [hunk 3]
```diff
@@ -279,4 +279,8 @@ object FunctionRegistry {
...
+  register("runif", { (max: Double) => max * math.random })
+  register("rnorm", { (mean: Float, sd: Float) => mean + sd * scala.util.Random.nextGaussian() })
```
Comment (cseed) [inline src/main/scala/org/broadinstitute/hail/expr/FunctionRegistry.scala]: Same.

File: src/main/scala/org/broadinstitute/hail/expr/FunctionRegistry.scala (hunk 2 above)
Comment (jbloom22) [inline src/main/scala/org/broadinstitute/hail/expr/FunctionRegistry.scala]: I switched to Float.

**Comment by jbloom22** [comment]
thanks, back to you

**Comment by tpoterba** [comment]
just a random comment: you don't need to amend commits anymore.  We can squash/commit instead of merging with github now

**Pull request merged**